packages/e2e-test-utils/src/drag-and-resize.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Clicks an element, drags a particular distance and releases the mouse button.
+ *
+ * @param {Object} element The puppeteer element handle.
+ * @param {number} delta   Object containing movement distances.
```

Comment:
Should this say `Object`?

```suggestion
 * @param {Object} delta   Object containing movement distances.
```

(http://usejsdoc.org is down for me right now and I can't remember if, for this type of argument, you're supposed to say `{Object}` or leave off the type altogether.)

Reply:
Yep, it was an oversight. Refactored the parameters a few times, and at some point introduced an error. I've committed the suggestion, thanks for spotting 😄